todo/src/reducers/index.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from "redux";
+import { itemReducer as items } from "./itemReducer";
```

Comment:
Since you refer to items so often in your TodoList component, maybe you should name this reducer something else? Like itemList or something.